test/forms/LoadingLocationFormProviderSpec.scala

```diff
@@ -62,20 +61,9 @@ class LoadingLocationFormProviderSpec extends StringFieldBehaviours with SpecBas
       )
     }
 
-    "when during transition" - {
-      val app = transitionApplicationBuilder().build()
-      running(app) {
-        val formProvider = app.injector.instanceOf[LoadingLocationFormProvider]
-        runTests(formProvider(prefix), 17)
-      }
-    }
-
     "when post transition" - {
-      val app = postTransitionApplicationBuilder().build()
-      running(app) {
-        val formProvider = app.injector.instanceOf[LoadingLocationFormProvider]
-        runTests(formProvider(prefix), 35)
-      }
+      val formProvider = new LoadingLocationFormProvider()
+      runTests(formProvider(prefix), 35)
```

Comment:
We can remove "when post transition" and remove runTests def to simplify everything here.